### DISCUSSION TITLE
Configurable track width for osu!mania

### DISCUSSION THREAD

**Discussion by RoboMico** [ppy/osu#18072] [Ideas] [answered]
When I played osu!mania on my tablet with touchscreen, I found that the tracks were too narrow to touch for fingers. I opened settings but found no configurable option for this, then I looked through the source code and it seems that the track width is defined as a const value:
[/osu.Game.Rulesets.Mania/UI/Column.cs, Line 29](https://github.com/ppy/osu/blob/04fb0f5e63d6dffbd2f3eba239b65ab6b698763d/osu.Game.Rulesets.Mania/UI/Column.cs#L29)
So why not make a option in settings, as there is a field `TrackWidth` in `skin.ini` in osu!stable skins, too. It can also be compatible with the skin for lazer in the futrue.

**Comment by peppy** [comment, marked as answer]
Yes, we will. Along with every other variable.
